core/src/main/kotlin/ch/ergon/dope/resolvable/clause/JoinOperator.kt

```diff
@@ -0,0 +1,53 @@
+package ch.ergon.dope.resolvable.clause
+
+import ch.ergon.dope.resolvable.expression.TypeExpression
+import ch.ergon.dope.resolvable.expression.unaliased.type.Field
+import ch.ergon.dope.resolvable.fromable.Bucket
+import ch.ergon.dope.validtype.BooleanType
+import ch.ergon.dope.validtype.ValidType
+
+sealed class JoinOperator : IFromClause {
```

Comment:
I think you should inherit from IJoinClause otherwise multiple joins are not possible, please add a test

Reply:
done